src/org/ods/orchestration/service/JiraService.groovy

```diff
@@ -793,4 +799,79 @@ class JiraService {
         }
         return new JsonSlurperClassic().parseText(response.getBody())
     }
+
+    protected List loadJiraSecurityVulnerabilityIssues(String issueSummary, String fixVersion,
+                                                       String jiraComponent, projectKey) {
+
+        def fields = ['assignee', 'duedate', 'issuelinks', 'status', 'summary']
+        def jql = "project = \"${projectKey}\" AND issuetype = \"Security Vulnerability\" " +
+            "AND fixVersion = \"${fixVersion}\" " +
+            "AND component = \"${jiraComponent}\" " +
+            "AND summary ~ \"${issueSummary}\" "
+
+        def jqlQuery = [
+            fields: fields,
+            jql: jql,
+            expand: []
+        ]
+
+        return getIssuesForJQLQuery(jqlQuery) ?: []
+    }
+
+    def getTransitions(issueId) {
```

Comment:
This will not be needed when we use the getIssue with transitions included.

Reply:
Removed.

Reply:
Fine now